integration_tests/base_routes.py

```diff
@@ -830,6 +830,85 @@ def sync_router_di(request, router_dependencies):
     return router_dependencies["ROUTER_DEPENDENCY"]
 
 
+# ===== Split request body =====
```

Comment:
Love the coverage! have some suggestions with the structure though 😄

Reply:
Please go ahead

Reply:
we should not use `jsonify` :D